rb/lib/selenium/webdriver/edge/legacy_support.rb

```diff
@@ -20,29 +20,27 @@
 module Selenium
   module WebDriver
     module Edge
-
       module LegacySupport
-
-          # These are commands Edge is still using from JSON Wire Protocol
+        # These are commands Edge is still using from JSON Wire Protocol
         %i(executeScript executeAsyncScript submitElement doubleClick mouseDown mouseUp mouseMoveTo
```

Comment:
oops. crap. Well, anyone using Edge shouldn't be using < 2.0, but I'll fix this in the 2.53 release... The downside of having coded with 2.0 for several years. :)